src/components/PersonLink.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  slug: string | null;
+  name: string | null;
+  sex: string | null;
+}
+
+export const PersonLink: React.FC <Props> = ({
+  slug,
+  name,
+  sex,
+}) => (
+  <>
+    {slug ? (
+      <Link
+        to={`/people/${slug}`}
+        className={classNames({
+          'has-text-danger': sex === 'f',
+        })}
+      >
+        {name}
+      </Link>
+    ) : (
+      <span>
+        {name || '-'}
```

Comment:
It would be cool not to use magic values and put such strings into constants and reuse them:
```
const FEMALE = 'f';
const NOT_SET_VALUE = '-';
```
This will increase the readability of the code and save us from possible errors.
This applies to development in general.